include/aie/Dialect/AIEX/Transforms/AIEXPasses.td

```diff
@@ -13,6 +13,23 @@
 
 include "mlir/Pass/PassBase.td"
 
+def AIEXToStandard : Pass<"aiex-standard-lowering", "ModuleOp"> {
```

Comment:
Tests are failing because after the `using namespace` cleanup, you need the mlir:: namespace here before the `ModuleOp`, like this:

```suggestion
def AIEXToStandard : Pass<"aiex-standard-lowering", "mlir::ModuleOp"> {
```